ai-chatbot/screens/SearchInChatroom/index.tsx

```diff
@@ -0,0 +1,10 @@
+import {View, Text} from 'react-native';
```

Comment:
we dont need this file, directly import form core, we used to give it when we pass `customSearchHeader`